client/landing/gutenboarding/components/domain-picker/button.tsx

```diff
@@ -65,28 +67,29 @@ const DomainPickerButton: FunctionComponent = () => {
 	const [ freeDomainSuggestions, paidDomainSuggestions ] = partition( suggestions, 'is_free' );
 
 	return (
-		<Button
-			className="domain-picker__button"
-			onClick={ () => setDomainPopoverVisibility( s => ! s ) }
-		>
-			<div className="domain-picker__site-title">
-				{ siteTitle ? siteTitle : NO__( 'Create your site' ) }
-			</div>
-			<div>{ head( freeDomainSuggestions )?.domain_name }</div>
+		<>
+			<Button
+				aria-expanded={ isDomainPopoverVisible }
+				aria-haspopup="menu"
+				aria-pressed={ isDomainPopoverVisible }
+				className={ classnames( 'domain-picker__button', { 'is-open': isDomainPopoverVisible } ) }
+				onClick={ () => setDomainPopoverVisibility( s => ! s ) }
+			>
+				{ /* This empty span gives us a good place to anchor our popover */ }
```

Comment:
Could otherwise wrap the free domain suggestion in the span? Might come in handy later if we need to style it.

Reply:
Apparently this didn't do anything, dropped it.